homeassistant/components/dynalite/panel.py

```diff
@@ -101,8 +102,8 @@ async def async_register_dynalite_frontend(hass: HomeAssistant):
     if DOMAIN not in hass.data.get("frontend_panels", {}):
         path = locate_dir()
```

Comment:
The check above need to be inside a lock.  Need to fix that first before moving this PR forward

Reply:
Instead I moved it to `async_setup` since we only want to do this once.  The websocket apis as well were being registered 2x